virtual_rainforest/models/soil/soil_model.py

```diff
@@ -42,9 +43,11 @@ class SoilModel(BaseModel):
 
     model_name = "soil"
     """The model name for use in registering the model and logging."""
+    required_init_vars = []
```

Comment:
How about using a type annotation here so that `mypy` knows what *kind* of list it will be?

Reply:
Does it not _have_ to follow the type of the abstract property in BaseModel? `mypy` certainly throws an error when it doesn't.

Having said that - it seems that we _have_ to manually add docstrings for the subclasses or these properties don't get documented. I think we probably could do something clever and make them inheritable (probably in `__init_subclass__` but they probably should be subclass specific.